modules/exploits/linux/local/hp_xglance_priv_esc.rb

```diff
@@ -169,13 +155,15 @@ def exploit
     # just to be safe, Xt and Xp were in the original exploit
     # our mock binary is also exploitsable through libXmu.so.6
     # unsure about the real binary
+    cd exploit_folder
     ['libXp.so.6', 'libXt.so.6', 'libXmu.so.6', lib].each do |l|
-      cmd_exec "cd #{exploit_folder}; ln -s libXm.so.3 #{l}"
+      cmd_exec "ln -s libXm.so.3 #{l}"
```

Comment:
Has this change been tested? You'll likely need to specify the full path.

```suggestion
      cmd_exec "ln -s libXm.so.3 #{exploit_folder}/#{l}"
```

Reply:
It looks like we `cd` into the exploit folder before launching these.  To be sure, I've tested it both ways, and neither seems to make a difference.